### ISSUE TITLE
this did not even work

### ISSUE THREAD

**Issue by grummbunger** [shystruk/SetIntervalJS#15]
i npm install with save. not global nor required as  you say import. i see the folder.

`
import  SetInterval from 'set-interval';

function drive(){
SetInterval.start(generateslow(), 1000, 'test');
 // tgvar = setInterval( function(){ generateslow(); },1000 );
}
`
the native syntax is here im comment.

**Comment by shystruk** [comment]
@grummbunger 
try this
```javascript
SetInterval.start(generateslow, 1000, 'test');
```
as the first parameter for method .start is a function. In your case, you are calling that function :)

**Comment by grummbunger** [comment]
ook so i solved  my issue. which was this. i have 4 apps in separate folders thinking that that would keep the global objects from each other. Turns out that on a node server with several hope the same application using the same variable globally here's the very same information with all the apps. So the problem wasn't that the clocks weren't stopping which is not my question, my question was actually why the program works on one system and not on the other. I needed to ask myself all the differences between where it was running properly and where it was not running properly. I had already tried versions of node as the version of node on my tablet is different than on my PC so now I have the same version on PC and tablet, but that was not the problem. At the same time the other object I was using is the timer object and the timer object is also shared globally it seems. Now at the very same time I also did not recognize that I have a routine that would also Auto respond like the clock but not using the clock in other words as a player makes his move a move is generated. But even with all these problems I was able to solve this the very next day. It had taken me a month to get that far. Having 40 years of experience programming means complex bugs.. Thanks for your response much appreciated.

⁣Get TypeApp for Android ​
